_includes/page_version.html

```diff
@@ -0,0 +1,5 @@
+{% if page.version %}
+  {% assign v = page.version %}
```

Comment:
Where is the `v` actually used? I don't see it even in `navigation.html` where it was used before.
Also a one letter variable, furthermore global it seems, that's rather obscuring.

Reply:
still in navigation

Reply:
actually can be put back

Reply:
> 
> 
> still in navigation

Ah I see. IMO would make better sense then to be defined somewhere and used elsewhere. Also what about a more explanatory name?

Reply:
Moved back